cdk/domino_cdk/config/eks.py

```diff
@@ -29,12 +29,15 @@ class NodegroupBase:
         key_name: some-key-pair - Pre-existing AWS key pair to configure for instances in the nodegorup
         min_size: 1 - Minimum node count for nodegroup. Can't be 0 on managed nodegroups.
         max_size: 10 - Maximum limit for node count in node gorup
-        machine_image: AMI and/or user_data script for nodes in nodegroup. When overriding AMI, you MUST
-                       setup your custom AMI or your user_data script _MANUALLY_ to join the cluster.
-                       Additionally, options that use user_data (ie ssm_agent) are no longer valid.
-                       machine_image:
-                         ami_id: ami-123abc
-                         user_data: ...
+        ami_id: ami-123abc - AMI to use for nodegroup, empty/null will use the default EKS AMI, which
+                             is the default. When specifying an AMI, you MUST specify a custom user_data
+                             script to join the node to the cluster, and this script must do any sort of
+                             node setup that is desired. Options that are implemented via user_data (ie
+                             labels, taints, ssm_agent) are not supported with custom AMIs, and must be
+                             implemented manualyl.
+        user_data: ... - Custom script for user_data, ran by cloud_init on node startup. When using the
+                         default EKS AMI, this will be ran in addition to the default user_data scripts.
```

Comment:
"in addition" -> "before"

Reply:
I didn't mean to suggest order, just indicate that your custom user data script doesn't replace the default one when you don't have a custom AMI. It's probably good to mention that it's before so people know what to expect, though, so I updated the language to convey both things.